components/cam/src/physics/cam/shoc.F90

```diff
@@ -1993,14 +1993,15 @@ subroutine shoc_assumed_pdf(&
   real(rtype) :: qw_sec_zt(shcol,nlev)
 
   ! define these so they don't have to be computed more than once
-  real(rtype), parameter :: sqrt2 = sqrt(2._rtype)
-  real(rtype), parameter :: sqrtpi = sqrt(2._rtype*3.14_rtype)
+  real(rtype) :: sqrt2, sqrtpi
+  sqrt2=bfb_sqrt(2._rtype)
+  sqrtpi=bfb_sqrt(2._rtype*3.14_rtype)
 
   epsterm=rgas/rv
 
   thl_tol=1.e-2_rtype
   rt_tol=1.e-4_rtype
-  w_tol_sqd=(2.e-2_rtype)**2
+  w_tol_sqd=bfb_square(2.e-2_rtype)
```

Comment:
Should `2.e` be changed to `2._rtype`?

Reply:
Yes.

Reply:
Actually, I think I read this incorrect. This is `2e-2`.